ksqldb-docker/README.md

```diff
@@ -13,7 +13,7 @@ To build a new image with local changes:
 
 1. Build docker images from local changes.
     ```
-    > mvn -Pdocker package -DskipTests -Dspotbugs.skip -Dcheckstyle.skip  -Ddockerfile.skip=false -Dskip.docker.build=false -Ddocker.upstream-tag=latest-ubi8 -Ddocker.tag=local.build  -Ddocker.upstream-registry=''
+    > mvn -Pdocker package -DskipTests -Dspotbugs.skip -Dcheckstyle.skip  -Ddockerfile.skip=false -Dskip.docker.build=false -Ddocker.upstream-tag=latest -Ddocker.tag=local.build  -Ddocker.upstream-registry=''
```

Comment:
Hang on, why are we undoing this change? Builds with the latest Dockerfile should fail with this change. Do they not for you?

Reply:
My bad - unintentional change.